src/lib/snack-bar/snack-bar-config.ts

```diff
@@ -0,0 +1,16 @@
+import {ViewContainerRef} from '@angular/core';
+
+
+export type SnackBarRole = 'alert' | 'polite';
+
+export class MdSnackBarConfig {
+  /** The aria-role of the snack bar. */
+  role: SnackBarRole = 'alert';
```

Comment:
`'polite'` isn't a valid `role` value. This should be used as the politeness that's passed through to the `MdLiveAnnouncer`